cse/reader/CommentReader.py

```diff
@@ -78,15 +77,15 @@ def __silentParseToInt(self, data, default):
 
 
     def __parseIterRow(self, row, skipArticleMapping=True):
-        commentId = int(row[0])
-        articleId = int(row[1])
+        commentId = int(row[2])
+        articleId = int(row[0])
         articleUrl = ""
-        author = row[2]
+        author = row[1]
         text = row[3].replace("\\n", "\n")
-        timestamp = row[4]
-        parentId = self.__silentParseToInt(row[5], None)
-        upvotes = self.__silentParseToInt(row[6], 0)
-        downvotes = self.__silentParseToInt(row[7], 0)
+        timestamp = row[5]
+        parentId = self.__silentParseToInt(row[4], None)
+        upvotes = 0 #self.__silentParseToInt(row[6], 0) # currently unused information
+        downvotes = 0 #self.__silentParseToInt(row[7], 0) # currently unused information
```

Comment:
this is the _the guardian_ mapping, isn't it?
Should we really change our implementation based on their broken format?

Reply:
already a todo. until everythin works...

Reply:
Added todo inline

Reply:
please don't forget this!